src/Containers/packaging/package.csproj

```diff
@@ -57,10 +59,10 @@
 
             <!-- tasks folder -->
             <Content Include="@(ContainerLibraryOutput)" Pack="true" PackagePath="tasks/$(TargetFramework)/" />
-            <Content Include="@(ContainerLibraryOutputNet472)" Pack="true" PackagePath="tasks/$(VSCompatTargetFramework)/" />
-            <Content Include="../Microsoft.NET.Build.Containers/bin/$(Configuration)/$(TargetFramework)/Microsoft.NET.Build.Containers.deps.json" Pack="true" PackagePath="tasks/$(TargetFramework)" />
+            <Content Include="@(ContainerLibraryOutputNet472)" Pack="true" PackagePath="tasks/$(VSCompatTargetFramework)/" Condition="'$(DotNetBuildFromSource)' != 'true'" />
+            <Content Include="../../../artifacts/bin/Microsoft.NET.Build.Containers/$(Configuration)/$(TargetFramework)/Microsoft.NET.Build.Containers.deps.json" Pack="true" PackagePath="tasks/$(TargetFramework)" />
```

Comment:
There are several hardcoded relative paths in this file.  Ideally it would be good to avoid that, but in any case I think it would be better if they were relative to `$(ArtifactsBinDir)`, `$(RepoRoot)`, or something similar instead of relative to the current project.

Reply:
Addressed in new change.